plugins/modules/ipavault.py

```diff
@@ -520,48 +531,57 @@ def main():
                     group_add, group_del = \
                         gen_add_del_lists(groups,
                                           res_find.get('member_group', []))
+                    service_add, service_del = \
+                        gen_add_del_lists(services,
+                                          res_find.get('member_service', []))
+
                     owner_add, owner_del = \
                         gen_add_del_lists(owners,
                                           res_find.get('owner_user', []))
+
                     ownergroups_add, ownergroups_del = \
                         gen_add_del_lists(ownergroups,
                                           res_find.get('owner_group', []))
 
                     # Add users and groups
-                    if len(user_add) > 0 or len(group_add) > 0:
+                    if len(user_add) > 0 or len(group_add) > 0 \
```

Comment:
Is it an issue to add empty lists with the commands? If yes, then you could add the empty list check also to `gen_member_args`. It would be needed only in there then.

Reply:
Modified to check lists on `gen_member_args()`.